s3auth-relay/pom.xml

```diff
@@ -174,6 +174,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
+            <version>3.3</version>
```

Comment:
if a project inherits from jcabi parent, you don't have to include versions for common libraries (look at `dependencyManagement` section in https://github.com/jcabi/jcabi-parent/blob/master/pom.xml)